internal/backend/sessions/sessionworkflows/metrics.go

```diff
@@ -0,0 +1,35 @@
+package sessionworkflows
+
+import (
+	"context"
+	"fmt"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/telemetry"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.opentelemetry.io/otel/metric"
+)
+
+var sessionsCounter metric.Int64UpDownCounter
+
+func initMetrics(t *telemetry.Telemetry) {
+	sessionsCounter = t.NewOtelUpDownCounter("sessions.gaude", "The session counter")
```

Comment:
gaude is not a word I think, unless I miss something

the type of the counter is gauge

it's not really a part of the name of the metric
the metric name should be something like autokitteh.active_sessions (which is what I understand from the up down counter)
otherwise autokitteh.total_sessions

Reply:
rewritten to use 4 counters